modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/search/test/DLFileEntrySearchByStatusTest.java

```diff
@@ -58,20 +58,6 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
```

Comment:
about removing the tests.
maybe changing them are more efficient than remove?
```

@Test	
	public void testSearchFileAnyStatus() throws Exception {	
		String titlePrefix = "Document";	

		_addFileEntry(null, titlePrefix + StringUtil.randomString());	

		Date displayDate = new Date(System.currentTimeMillis() + Time.DAY);	

		_addFileEntry(	
			displayDate, titlePrefix + " " + StringUtil.randomString());	

		_assertHits(1, titlePrefix, true, WorkflowConstants.STATUS_ANY);	
	}

```
...

	
```
		String title = "Document";	

		_addFileEntry(	
			new Date(System.currentTimeMillis() + Time.DAY),	
			title + " " + StringUtil.randomString());	

		_assertHits(0, title, true, WorkflowConstants.STATUS_ANY);
```